docs/running-on-kubernetes.md

```diff
@@ -394,6 +394,13 @@ spark.kubernetes.executor.volumes.persistentVolumeClaim.spark-local-dir-1.mount.
 spark.kubernetes.executor.volumes.persistentVolumeClaim.spark-local-dir-1.mount.readOnly=false
 ```
 
+To enable shuffle data recovery feature via the built-in `KubernetesLocalDiskShuffleDataIO` plugin, we need to have the followings. You may want to enable `spark.kubernetes.driver.waitToReusePersistentVolumeClaim` additionally.
```

Comment:
I only wonder this shuffle data recovery feature is not mentioned in this doc except here. Is it something we need to add link to or add more description?

Reply:
Yes, this is the only place to mention `KubernetesLocalDiskShuffleDataIO` plugin although `### PVC-oriented executor pod allocation` section has all PVC-related configurations. It's because the recovery feature is implemented by a plugin and used by `spark.shuffle.sort.io.plugin.class`.